packages/manager/modules/anthos/src/dashboard/general-information/general-information.html

```diff
@@ -43,9 +43,38 @@
                             class="mb-0"
                             data-ng-bind=":: $ctrl.tenant.version"
                         ></p>
+                        <div
+                            data-ng-if="$ctrl.isSoftwareUpdatable && !$ctrl.isSoftwareUpdating"
+                        >
+                            <p>
+                                <span
+                                    class="oui-badge oui-badge_warning"
+                                    data-translate="anthos_tenant_dashboard_general_information_tile_information_software_is_updatable"
+                                ></span>
+                            </p>
+                            <button
+                                type="button"
+                                class="oui-button oui-button_s oui-button_ghost"
+                                data-ng-click="$ctrl.goToSoftwareUpdate()"
+                            >
+                                <span
+                                    data-translate="anthos_tenant_dashboard_general_information_tile_information_software_update_link"
+                                ></span>
+                                <span
+                                    class="oui-icon oui-icon-arrow-right"
+                                    aria-hidden="true"
+                                ></span>
+                            </button>
+                        </div>
                         <p
+                            data-ng-if="$ctrl.isSoftwareUpdating"
                             class="oui-badge oui-badge_warning"
-                            data-translate="anthos_tenant_dashboard_general_information_tile_information_software_status_upcoming"
+                            data-translate="anthos_tenant_dashboard_general_information_tile_information_software_is_updating"
+                        ></p>
+                        <p
+                            data-ng-if="!$ctrl.isSoftwareUpdatable"
+                            class="oui-badge oui-badge_info"
```

Comment:
**question:** L20, you are displaying a badge via a `span` (inline) element. Why are you using here a `p` (block) element?